instrumentation/google.golang.org/grpc/otelgrpc/test/interceptor_test.go

```diff
@@ -583,39 +583,150 @@ func TestStreamClientInterceptorWithError(t *testing.T) {
 	assert.Equal(t, codes.Error, span.Status().Code)
 }
 
-func TestServerInterceptorError(t *testing.T) {
+var serverChecks = []struct {
+	grpcCode grpc_codes.Code
+	spanCode codes.Code
```

Comment:
Can we rename it to `wantSpanCode` to indicate that this is what we are expecting to get?